tests/upgrades/test_errata.py

```diff
@@ -103,7 +101,9 @@ class TestScenarioErrataCount(TestScenarioErrataAbstract):
     """
 
     @pytest.mark.pre_upgrade
-    def test_pre_scenario_generate_errata_for_client(self, save_test_data):
+    def test_pre_scenario_generate_errata_for_client(
```

Comment:
AFAIK, w/o katello-agent, errata can be applied through REX (Job category = Katello, Job template = Install Errata - Katello Script Default)

Reply:
We are leaving this for @vsedmik Team to move this scenario to the REX based because:
- We are running out of time to do this scenario refactor. 
- The component-specific improvements is not part of upgrade scenarios refactor
- The Teams can better do the component specific refactor as experts than any JPL member.